provider/lxd/credentials.go

```diff
@@ -174,10 +216,19 @@ func (p environProviderCredentials) FinalizeCredential(ctx environs.FinalizeCred
 		// $HOME/.config/lxc/config.yml to identify the remote by its
 		// endpoint.
 		//
-		// We may later want to add an "interactive" auth-type
-		// that users can select when adding credentials to
-		// go through the interactive server certificate fingerprint
-		// verification flow.
+		// TODO(axw) for the "interactive" auth-type, we should take
+		// the user through the server certificate fingerprint
+		// verification and trust password flow.
+		prefix := "cannot auto-generate credential for remote LXD"
+		if interactive {
+			prefix = "certificate upload for remote LXD unsupported"
+		}
+		return nil, errors.Errorf(`%s
+
+Until support is added for verifying and authenticating to remote LXD hosts,
+you must generate the credential by hand, adding the certificate to LXD using
+the "lxc config trust" command.
```

Comment:
Can we point the user to online doc here, just like we do in other cases, eg how to properly configure lxd kernel params.
The text tells to user something needs to be done but doesn't show them how etc. Online we can walk them through the steps. When to run autoload-credential? When to choose interactive vs certificate etc. Describe the main usage scenarios.